### PR TITLE
[max] Improve German translations slightly, resolve ambiguities

### LINKED ISSUE #4605: [max] Improve texts
## Expected Behavior
Texts should not be ambiguous. :-)

## Current Behavior
Some texts are ambiguous or not easy to understand.

See discussion in https://community.openhab.org/t/solved-max-what-is-thermostat-window-open-duration/61343 plus a few more minor text problems.

## Possible Solution
Improve texts

## Steps to Reproduce (for Bugs)
View advanced settings of the binding

## Context
It's confusing the user. At least, it confused me. 

## Your Environment
openHAB 2.4.0

### PR DESCRIPTION
**Pull request by cubistico**
Signed-off-by: Metin Savignano ms@savignano.net (github: cubistico)

Just improved some German descriptions:

- Replaced "Zeigt an" by "Gibt an" which appears more appropriate to me
- Fixed typos and grammatics 
- Clarified the window open duration description

This PR replaces the previous one.

Fixes #4605

### CODE REVIEW

File: addons/binding/org.openhab.binding.max/ESH-INF/i18n/max_de.properties [hunk 1]
```diff
@@ -129,25 +129,25 @@ thing-type.config.max.thermostat.name.label = Name
...
 
 thing-type.config.max.thermostat.comfortTemp.label = Komforttemperatur
-thing-type.config.max.thermostat.comfortTemp.description = Zeigt die aktuell eingestellte Komforttemperatur (in �C) des Heizk�rperthermostat an.
+thing-type.config.max.thermostat.comfortTemp.description = Gibt die Komforttemperatur (in �C) des Heizk�rperthermostats an.
 
 thing-type.config.max.thermostat.ecoTemp.label = Absenktemperatur
-thing-type.config.max.thermostat.ecoTemp.description = Zeigt die aktuell eingestellte Absenktemperatur (in �C) des Heizk�rperthermostat an.
+thing-type.config.max.thermostat.ecoTemp.description = Gibt die Absenktemperatur (in �C) des Heizk�rperthermostats an.
 
 thing-type.config.max.thermostat.offsetTemp.label = Temperatur Offset
```
Comment (cweitkamp) [inline addons/binding/org.openhab.binding.max/ESH-INF/i18n/max_de.properties]: We should use the same spelling like in the description: "Temperatur-Offset"

File: addons/binding/org.openhab.binding.max/ESH-INF/i18n/max_de.properties [hunk 2]
```diff
@@ -169,25 +169,25 @@ thing-type.config.max.thermostatplus.name.label = Name
...
 
 thing-type.config.max.thermostatplus.comfortTemp.label = Komforttemperatur
-thing-type.config.max.thermostatplus.comfortTemp.description = Zeigt die aktuell eingestellte Komforttemperatur (in �C) des Heizk�rperthermostat an.
+thing-type.config.max.thermostatplus.comfortTemp.description = Gibt die Komforttemperatur (in �C) des Heizk�rperthermostats an.
 
 thing-type.config.max.thermostatplus.ecoTemp.label = Absenktemperatur
-thing-type.config.max.thermostatplus.ecoTemp.description = Zeigt die aktuell eingestellte Absenktemperatur (in �C) des Heizk�rperthermostat an.
+thing-type.config.max.thermostatplus.ecoTemp.description = Gibt die Absenktemperatur (in �C) des Heizk�rperthermostats an.
 
 thing-type.config.max.thermostatplus.offsetTemp.label = Temperatur Offset
```
Comment (cweitkamp) [inline addons/binding/org.openhab.binding.max/ESH-INF/i18n/max_de.properties]: see above

File: addons/binding/org.openhab.binding.max/ESH-INF/i18n/max_de.properties [hunk 3]
```diff
@@ -169,25 +169,25 @@ thing-type.config.max.thermostatplus.name.label = Name
...
-thing-type.config.max.thermostatplus.minTempSetpoint.description = Zeigt die minimale Solltemperatur (in �C) des Heizk�rperthermostat an.
+thing-type.config.max.thermostatplus.minTempSetpoint.description = Gibt die minimal einstellbare Solltemperatur (in �C) des Heizk�rperthermostats an.
 
 thing-type.config.max.thermostatplus.windowOpenTemp.label = Fenster-Auf-Temperatur
-thing-type.config.max.thermostatplus.windowOpenTemp.description = Zeigt die Temperatur bei ge�ffnetem Fenster-Zustand (in �C) des Heizk�rperthermostat an.
+thing-type.config.max.thermostatplus.windowOpenTemp.description = Gibt die Temperatur bei ge�ffnetem Fenster-Zustand (in �C) des Heizk�rperthermostats an.
 
 thing-type.config.max.thermostatplus.windowOpenDuration.label = Dauer Fenster-Auf-Temperatur
-thing-type.config.max.thermostatplus.windowOpenDuration.description = Zeigt die Dauer der Fenster-Auf-Temperatur (in �C) an.
+thing-type.config.max.thermostatplus.windowOpenDuration.description = Gibt die Dauer der Abseknung auf die Fenster-Auf-Temperatur (in Minuten) des Heizk�rperthermostats an.
```
Comment (cweitkamp) [inline addons/binding/org.openhab.binding.max/ESH-INF/i18n/max_de.properties]: Spelling: "Absenkung"

File: addons/binding/org.openhab.binding.max/ESH-INF/i18n/max_de.properties [hunk 4]
```diff
@@ -129,25 +129,25 @@ thing-type.config.max.thermostat.name.label = Name
...
 
 thing-type.config.max.thermostat.ecoTemp.label = Absenktemperatur
-thing-type.config.max.thermostat.ecoTemp.description = Zeigt die aktuell eingestellte Absenktemperatur (in �C) des Heizk�rperthermostat an.
+thing-type.config.max.thermostat.ecoTemp.description = Gibt die Absenktemperatur (in �C) des Heizk�rperthermostats an.
 
 thing-type.config.max.thermostat.offsetTemp.label = Temperatur Offset
-thing-type.config.max.thermostat.offsetTemp.description = Zeigt die aktuell eingestellten Temperatur Offset (in �C) des Heizk�rperthermostat an.
+thing-type.config.max.thermostat.offsetTemp.description = Gibt den Temperatur-Offset (in �C) des Heizk�rperthermostats an.
 
 thing-type.config.max.thermostat.maxTempSetpoint.label = Max. Temperatur
```
Comment (cweitkamp) [inline addons/binding/org.openhab.binding.max/ESH-INF/i18n/max_de.properties]: Should we change the labels accordingly? "Max. Solltemperatur"

File: addons/binding/org.openhab.binding.max/ESH-INF/i18n/max_de.properties [hunk 5]
```diff
@@ -250,29 +250,29 @@ thing-type.config.max.shuttercontact.action-deviceDelete.option.-1 = Keine Aktio
...
-channel-type.max.mode.description = Zeigt den aktuellen Modus des Ger�tes an (MANUAL/AUTOMATIC/BOOST/VACATION).
+channel-type.max.mode.description = Gibt den aktuellen Modus des Ger�tes an (MANUAL/AUTOMATIC/BOOST/VACATION).
 channel-type.max.mode.state.option.AUTOMATIC = Automatisch
 channel-type.max.mode.state.option.MANUAL = Manuell
 channel-type.max.mode.state.option.BOOST = Boost
 channel-type.max.mode.state.option.VACATION = Urlaubsmodus
 
 channel-type.max.actual_temp.label = Temperatur
-channel-type.max.actual_temp.description = Zeigt die aktuell gemessene Temperatur des Heizk�rperthermostat an.
+channel-type.max.actual_temp.description = Gibt die aktuell gemessene Temperatur (in �C) des Heizk�rperthermostats an.
```
Comment (cweitkamp) [inline addons/binding/org.openhab.binding.max/ESH-INF/i18n/max_de.properties]: Please remove the unit. It depends on the users configuration as the channel uses QuantityType.

File: addons/binding/org.openhab.binding.max/ESH-INF/i18n/max_de.properties [hunk 6]
```diff
@@ -250,29 +250,29 @@ thing-type.config.max.shuttercontact.action-deviceDelete.option.-1 = Keine Aktio
...
 
 channel-type.max.set_temp.label = Solltemperatur
-channel-type.max.set_temp.description = Erm�glicht die Steuerung der Solltemperatur des Heizk�rperthermostat.
+channel-type.max.set_temp.description = Gibt die aktuell eingestellte Solltemperatur (in �C) des Heizk�rperthermostats.
```
Comment (cweitkamp) [inline addons/binding/org.openhab.binding.max/ESH-INF/i18n/max_de.properties]: Please remove the unit. It depends on the users configuration as the channel uses QuantityType.

**Comment by cubistico** [comment]
I've got a general questions wrt the workflow: If I apply a requested change, should I resolve the comment or should it be kept as a reminder for the reviewer?

**Pull request merged**